fedimint-core/src/db/mod.rs

```diff
@@ -339,6 +339,40 @@ impl Database {
     }
 }
 
+#[apply(async_trait_maybe_send!)]
+pub trait IDatabaseTransactionOps<'a>: 'a + MaybeSend {
```

Comment:
Basically I extracted this, so this methods can't diverge between multiple traits.